src/renderers/dom/test/__tests__/ReactTestUtils-test.js

```diff
@@ -241,7 +241,7 @@ describe('ReactTestUtils', () => {
     );
   });
 
-  it('should throw when attempting to use ReactTestUtils.Simulate with shallow rendering', () => {
+  it('should throw when attempting to use a React element', () => {
```

Comment:
The old test name was clearer. The new test name doesn't explain *what* should throw. Similarly "attempting to use a React element" is very vague.

I think the choice here is between the old name and "should throw when attempting to pass React element to Simulate". I like the old name better because it reflects a real use case where this could happen.